### PR TITLE
Remove query interface from spec

### PR DESCRIPTION
**Pull request by OR13**
Addresses concerns about HTTP / interface portability raised on https://github.com/w3c-ccg/universal-wallet-interop-spec/issues/46

### CODE REVIEW

**Review by mprorock — APPROVED** [review]

**Review by kimdhamilton — APPROVED** [review]

File: docs/index.html [hunk 1]
```diff
@@ -1138,12 +1138,22 @@ <h2>Interface</h2>
         as input, and they produce wallet contents and side effects as output.
       </p>
 
+      <p>
+        In order to ensure portability, wallet interfaces MUST be functions of
+        JSON, and not take complex instances of classes, functions or other
+        types of object.
```
Comment (TallTed) [inline docs/index.html:1144]:
```suggestion
In order to ensure portability, wallet interfaces MUST be functions of
JSON, and not take complex instances of classes, functions, or other
types of object.
```

File: docs/index.html [hunk 2]
```diff
@@ -1138,12 +1138,22 @@ <h2>Interface</h2>
...
+
       <section>
         <h3>Import</h3>
         <p>
           Takes a serialized exported wallet representation as input. Loads the
           representation into wallet software.
         </p>
+        <p>
+          This method may not be necessary, for implementations that rely on
+          external storage, such as Encrypted Data Vaults.
```
Comment (TallTed) [inline docs/index.html:1155]:
```suggestion
This method may not be necessary for implementations that rely on
external storage, such as Encrypted Data Vaults.
```

File: docs/index.html [hunk 3]
```diff
@@ -1153,6 +1163,11 @@ <h3>Unlock</h3>
         </p>
 
         <p>Requires knowledge of the password used to lock the wallet.</p>
+
+        <p>
+          This method may not be necessary, for implementations that rely on
+          external storage, such as Encrypted Data Vaults.
```
Comment (TallTed) [inline docs/index.html:1169]:
```suggestion
This method may not be necessary for implementations that rely on
external storage, such as Encrypted Data Vaults.
```

File: docs/index.html [hunk 4]
```diff
@@ -1162,21 +1177,17 @@ <h3>Lock</h3>
         </p>
         <p>Requires knowledge of the password used to lock the wallet.</p>
 
+        <p>
+          This method may not be necessary, for implementations that rely on
+          external storage, such as Encrypted Data Vaults.
```
Comment (TallTed) [inline docs/index.html:1182]:
```suggestion
This method may not be necessary for implementations that rely on
external storage, such as Encrypted Data Vaults.
```

**Review by TallTed — COMMENTED** [review]
Motivated by punctuation; also touched whitespace

**Pull request merged**